test/tools/ossfuzz/protoToAbiV2.h

```diff
@@ -745,9 +745,14 @@ class AssignCheckVisitor: public AbiV2ProtoVisitor<std::pair<std::string, std::s
 	{
 		ValueStream() = default;
 		unsigned index = 0;
+		unsigned prevIndex = 0;
```

Comment:
Is that actually used or just leftover? Doesn't look like it's used :-).
Apart from this, looks good!

Reply:
Ah right! Fixed it now